script.js

```diff
@@ -0,0 +1,96 @@
+// send login details
+function sendLoginDetails() {
+    var userName = document.getElementById("userName");
```

Comment:
Change the identity of this "username" component to "email"

Reply:
Done.